pkg/etcdutil/etcdutil_test.go

```diff
@@ -42,6 +42,8 @@ func newTestSingleConfig() *embed.Config {
 	cfg.Name = "test_etcd"
 	cfg.Dir, _ = ioutil.TempDir("/tmp", "test_etcd")
 	cfg.WalDir = ""
+	cfg.Logger = "zap"
```

Comment:
do we use zap in production?

Reply:
etcd plan to use [`zap`](https://github.com/uber-go/zap) replace the `capnslog` logging. and we enable it in the test.